src/pages/Article/ArticlePage.jsx

```diff
@@ -78,5 +64,10 @@ function ArticlePage({ article }) {
     </PageLayout>
   );
 }
+ArticlePage.propTypes = {
+  title: PropTypes.string,
+  content: PropTypes.string,
+  link: PropTypes.string,
+};
```

Comment:
Your props are not required, but you do not provide default values

Reply:
fixed